docs/source/pages/howToWorkWithPlatform/adminApplication/users/users.rst

```diff
@@ -2,173 +2,191 @@
 
-.. image:: pic_users/roles.png
+.. image:: pic_users/reportRolesTab.png
    :width: 400
    :align: center
 
-2. Second sheet is "Permissions". On this sheet are showing permissions which can be used on the Account and Mailbox list
+2. Second sheet is "Permissions". It lists all permissions which can be used on the "Account" and "Mailbox" sheets
 
-.. image:: pic_users/permissions.png
+.. image:: pic_users/reportPermissionsTab.png
    :width: 400
    :align: center
 
-3. Third sheet is "Users". There are we have to fill users data (first name, last name and email). Also here you can choose to create or not personal user's mailbox (fill Y or N (or leave it empty) for create or not create). According to theese data users would be created
+3. Third sheet is "Users". Here you should enter users data (first name, last name and email). Also you can choose should a personal mailbox be created for each user individually (Y to create, N or leave empty to not create). Users will be created according to this data
+
+.. warning:: All users in the list should have relations to an active domain(s). If any users without such relation will be added to the list, user creation process will fail.
+
+.. image:: pic_users/reportUsersTab.png
+   :width: 400
+   :align: center
 
-.. warning:: All users in the list have to have relation to verified domain(s). So if you decided to add user to the list from unverified or inactive domains, all process of users creation will be stoped.
+4. Fourth sheet is "Account". This sheet allows to set account roles and permissions for each user from the list
 
-.. image:: pic_users/users.png
+.. image:: pic_users/reportAccountTab.png
    :width: 400
    :align: center
 
-4. Fourth sheet is "Account". This sheet is serves to set access levels to account for particular users from the list users. You can set access to account using roles or permissions
+.. warning:: If any incorrect emails will be added to the sheet, such users will not be created.
 
-.. warning:: If you add incorrect email to the sheet it will not be added to the platform.
+5. Fifth and the rest of sheets are "Mailbox" sheets. These sheets allow to set mailbox roles and permissions for each user from the list
 
-.. image:: pic_users/accountAccess.png
+.. image:: pic_users/reportMailboxTab.png
    :width: 400
    :align: center
 
-5. Fifth and the next sheets are "Mailboxes". Each sheet has identification of the mailbox in the top of the sheet. You can set access to mailbox using roles or permissions
+.. warning:: If any incorrect emails will be added to the sheet, such users will not be created.
 
-.. warning:: If you add incorrect email to the sheet it will not be added to the platform.
+When you will fill the file with data you should upload it in the users creation form and submit it. All users will receive email notifications with the instructions as described below.
```

Comment:
When you have filled the file with data, you should upload it in the user creation form and submit it.